plugins/CoreHome/vue/src/WidgetLoader/WidgetLoader.vue

```diff
@@ -253,6 +253,11 @@ export default defineComponent({
           this.loadingFailedRateLimit = true;
         }
 
+        if (response.status === 401) {
+          Matomo.helper.redirect();
```

Comment:
using `refreshAfter(0)` might be more suitable here. redirecting to a page with the same url and hash might result in doing nothing.